week1/armstrong.rb

```diff
@@ -9,5 +9,26 @@
 # and `false` otherwise.
 
 def armstrong_number(n)
-    raise NotImplementedError # Delete this line and start coding here
-end
+    num = n
+    temp = num
+    digits = 0
+    sum = 0
+
+    while (temp>0)
+	    digits+=1
+	    temp=temp/10
+    end
+
+    temp = num
+
+    while (temp > 0)
+        sum += (temp%10)**digits
+        temp = temp/10
+    end
```

Comment:
You can very easily find the digits of a number:
```ruby
array_of_digits = n.digits
```

Reply:
Ah yes! Thanks!